lib/evmone/eof.cpp

```diff
@@ -3,27 +3,54 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "eof.hpp"
+#include "baseline_instruction_table.hpp"
 #include "instructions_traits.hpp"
 
+#include <algorithm>
 #include <array>
 #include <cassert>
 #include <limits>
+#include <numeric>
+#include <stack>
 #include <variant>
+#include <vector>
 
 namespace evmone
 {
 namespace
 {
 constexpr uint8_t MAGIC[] = {0xef, 0x00};
 constexpr uint8_t TERMINATOR = 0x00;
-constexpr uint8_t CODE_SECTION = 0x01;
-constexpr uint8_t DATA_SECTION = 0x02;
+constexpr uint8_t TYPE_SECTION = 0x01;
+constexpr uint8_t CODE_SECTION = 0x02;
+constexpr uint8_t DATA_SECTION = 0x03;
 constexpr uint8_t MAX_SECTION = DATA_SECTION;
+constexpr auto CODE_SECTION_NUMBER_LIMIT = 1024;
+constexpr auto MAX_STACK_HEIGHT = 0x0400;
```

Comment:
In unified spec it's
```suggestion
constexpr auto MAX_STACK_HEIGHT = 0x03FF;
```

Reply:
Done